cms/djangoapps/contentstore/views/course.py

```diff
@@ -871,6 +878,10 @@ def _create_or_rerun_course(request):
         fields = {'start': start}
         if display_name is not None:
             fields['display_name'] = display_name
+        if language is not None:
+            fields['language'] = language
+        if is_basic_rerun is None:
```

Comment:
Yes, now we are passing language with course create requests. In case we don't pass language it automatically set language to the default language English.

Reply:
I was asking about line "883"
`if is_basic_rerun is None:` check.